lms/djangoapps/courseware/module_render.py

```diff
@@ -430,7 +439,7 @@ def handle_grade_event(block, event_type, event):  # pylint: disable=unused-argu
         grade = event.get('value')
         max_grade = event.get('max_value')
 
-        field_data_cache.set_score(
+        set_score(
```

Comment:
No, we should only be sending grades for the current user. However, I think that assertion will have to live outside the `set_score` function, since that function doesn't have any way of knowing who the current user is.